lib/matplotlib/__init__.py

```diff
@@ -1445,3 +1445,8 @@ def inner(ax, *args, data=None, **kwargs):
 _log.debug('interactive is %s', is_interactive())
 _log.debug('platform is %s', sys.platform)
 _log.debug('loaded modules: %s', list(sys.modules))
+
+
+# workaround: we must defer colormaps import to after loading rcParams, because
+# colormap creation depends on rcParams
+from matplotlib.cm import _colormaps as colormaps
```

Comment:
Could we handle this in the module level `__getattr__` now that we have than in place?